### PR TITLE
Adding Calendar-Wikivoyage Extension

### PR DESCRIPTION
**Pull request by examknow**
Adding Calendar-Wikivoyage extension per phab item T5137. Submodule has not yet been added. I will add comment when it is.

### CODE REVIEW

File: LocalExtensions.php [hunk 1]
```diff
@@ -237,6 +237,10 @@
 	wfLoadExtension( 'CSS' );
 }
 
+if ( $wmgUseCalendarWikivoyage ) {
+	wfLoadExtension( 'Calendar-Wikivoyage' )
```
Comment (Reception123) [inline LocalExtensions.php:241]: Please use the original name (https://github.com/wikimedia/mediawiki-extensions-Calendar/blob/master/extension.json#L2) for this one or else it won't work

Comment (examknow) [reply LocalExtensions.php:241]: I will note that the mediawiki.org docs for this extension are incorrect
image

**Comment by examknow** [comment]
Submodule added. Ready to merge when you are 😃.

File: LocalExtensions.php (hunk 1 above)
Comment (RhinosF1) [reply LocalExtensions.php]: https://www.mediawiki.org/w/index.php?title=Topic:Vlax350jbs1yysjz&action=history

**Pull request merged**